packages-nextgen/kestrel_core/src/kestrel/frontend/mapping/property/stix.yaml

```diff
@@ -0,0 +1,196 @@
+autonomous-system:name: organization.name
+autonomous-system:number: organization.uid
+
+directory:path: file.path
+directory:accessed: file.accessed_time
+directory:created: file.created_time
+directory:modified: file.modified_time
+
+domain-name.value:
```

Comment:
Potential an issue for data translation. See `file.hashes` for detailed example.